README.md

```diff
@@ -4,8 +4,8 @@ New Tab page in Ghostery Dawn is implemented as web-extension.
 
 ## Development
 
-Start extension in local Ghostery Dawn installation:
+Start extension:
 
 ```
-npm run start -- --firefox /Applications/Ghostery\ Browser.app/Contents/MacOS/Ghostery
```

Comment:
would be nice to keep info about Ghostery Private Browser

Reply:
I added a new default target as the Ghostery Browser, with the above path.